js/main.js

```diff
@@ -1,13 +1,15 @@
 import {loadTodos} from './todo.js'
 import {showTime} from './clock.js'
 import {isUsernameSet,loadUsername} from './userGreeting.js'
+import {loadBackgroundColor} from './changeColor.js'
 
 // so that we can use anywhere, will be used for pretty much everything
 export let storage = window.localStorage;
 
 document.addEventListener("DOMContentLoaded", function() {
   loadTodos();
   showTime();
+  loadBackgroundColor();
```

Comment:
A future not for us, is how we will solve this when the user can choose what parts that are enabled, but that is not relevant to this PR

Reply:
A new module that keeps track of all the settings :D